### PR TITLE
fixed the put token validate error

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java [hunk 1]
```diff
@@ -37,38 +32,36 @@ public AuthInterceptor(String allowSourceCollection, TokenService tokenService)
...
 			}
 		}
 		return true;
 	}
 
-	private boolean validateCspToken(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
-		final String token = request.getHeader(ConstantsKeys.CSP_AUTH_TOKEN);
+	private boolean validateCspToken(final String token, final HttpServletResponse response) throws IOException {
 		if(token == null) {
+			response.setContentType("application/json;charset=utf-8");
```
Comment (linr211) [inline g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java:58]: use the string in ConstantsKeys.CONTENT_TYPE_JSON?

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java [hunk 2]
```diff
@@ -37,38 +32,36 @@ public AuthInterceptor(String allowSourceCollection, TokenService tokenService)
...
 			response.setStatus(HttpStatus.UNAUTHORIZED.value());
 			response.getWriter().write(this.buildRespBody(HttpStatus.UNAUTHORIZED.value(), ConstantsKeys.TOKEN_VALIDATION_ERROR));
 			return false;
 		}
 		if (!tokenService.isTokenValid(token)) {
-			// The user is not authenticated.
+			response.setContentType("application/json;charset=utf-8");
```
Comment (linr211) [inline g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java:64]: use the string in ConstantsKeys.CONTENT_TYPE_JSON?

Comment (tigershi) [reply g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java]: done

File: g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/auth/interceptor/AuthInterceptor.java]: done

**Review by linr211 — APPROVED** [review]

**Review by jin-jasper — APPROVED** [review]

**Pull request merged**